code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -403,7 +403,7 @@
 	return TRUE
 
 /obj/item/borg/upgrade/traitorsec
-	name = "Bloodhound upgrade module"
+	name = "Opressor upgrade module"

```

Comment:
The correct spelling is "Oppressor". You need two "p"s.

Reply:
Don't lecture me on your Burgerstan language!